polygeist/lib/Dialect/Polygeist/Transforms/RaiseToAffine.cpp

```diff
@@ -41,24 +42,10 @@ struct RaiseSCFToAffine
 };
 } // namespace
 
+/// Attempt to transform an scf.ForOp loop to an AffineForOp loop.
```

Comment:
```suggestion
/// Attempt to transform an scf.for loop to an affine.for loop.
```
or 
`/// Attempt to transform an scf::ForOp loop to an affine::ForOp loop.`

Reply:
Not going to commit this one, is not worth having to rerun the tests.